### ISSUE TITLE
SyntaxError on your example code

### ISSUE THREAD

**Issue by LocutusOfPenguin** [amasend/lichees_python_SDK#36]
Trying your example:
from lichess_client import APIClient
client = APIClient(token="lichess_account_token")
response = await client.account.get_my_profile()

gives me a 

>>> response = await client.account.get_my_profile()
  File "<stdin>", line 1
    response = await client.account.get_my_profile()
                          ^
SyntaxError: invalid syntax

error message. I use python3.6.9

BTW. in your readme.md its "lichees" written, so please update to "lichess"

**Comment by gbtami** [comment]
You need an event loop to use await.
```
import asyncio
from lichess_client import APIClient

async def main():
    client = APIClient(token="your_lichess_account_token")
    response = await client.account.get_my_profile()
    print(response)

loop = asyncio.get_event_loop()
asyncio.ensure_future(main())
loop.run_forever()
loop.close()
```

**Comment by amasend** [comment]
1. @gbtami example is correct, also you can pass an event loop directly to APIClient:
```
import asyncio
from lichess_client import APIClient

loop = asyncio.get_event_loop()
client = APIClient(token="your_lichess_account_token", loop=loop)
```
2. Typo in Readme corrected.

**Comment by LocutusOfPenguin** [comment]
thanks @gbtami 
yes, updating the readme would be great (to help people like me, aehm).
I get a "unclosed connector" message , even i change "run_forever" to "run_until_complete"

**Comment by LocutusOfPenguin** [comment]
small update
@amasend sorry for being too small-minded/picky.
In your (great!) docu for the async for-loops y missing a ":" everywhere

**Comment by amasend** [comment]
@LocutusOfPenguin To be more precise, from version 7.x of IPython there is a possibility to call `await` directly in the notebook cell, without a need to wrap it with a main coroutine and calling a loop from the top level [IPython 7.0, Async REPL](https://blog.jupyter.org/ipython-7-0-async-repl-a35ce050f7f7). This could be misleading example from my side as someone probably like you, want to call these methods from the pure Python interpreter. I will update examples in Readme, also thanks for reading auto-generated doc, there will be an update for these missing colons.

Regaring `unclosed connector`, I've changed how session is handled right now, so there should not be any issue with this one.

PR: https://github.com/amasend/lichess_python_SDK/pull/38

**Comment by amasend** [comment]
Closing as this is implemented in version 1.1.0.6.
